### DISCUSSION TITLE
Quotes Page

### DISCUSSION THREAD

**Discussion by Accio1** [theforumhelpers/theforumhelpers.github.io#10] [Ideas]
How should the quote/funny topic page look?

I was thinking to have it set up similar to the list of Forum Helpers page, but I want to hear everyone else's thoughts as well.

**Comment by NilsTheBest** [comment]
I can't really think of anything else either, we might just stick with that until we find something original

**Comment by NilsTheBest** [comment]
Is there any way I could help though? I'm looking forwards to contribute to the making of it, and not do nothing :P

**Comment by Accio1** [comment]
Sure! What would you like to help with?

Step one is obviously gathering quotes, and deciding on a layout.

**Comment by NilsTheBest** [comment]
"By any chance are you using Safari or Chrome browser? There's a known issue where using either one of those two browsers result in **demon robot voice output**. " - Cheddargirl, 2019
https://scratch.mit.edu/discuss/post/3536182

**Comment by Accio1** [comment]
I made a mockup of what the quotes page could look like. 

Thoughts?

![Screenshot 2019-06-15 at 8 52 55 PM](https://user-images.githubusercontent.com/48496940/59557744-9eeea700-8faf-11e9-8c21-54e1b2fd484a.png)

**Comment by NilsTheBest** [comment]
That looks pretty accurate! I don't really think we need the two categories though, you could just have "iconic quotes", and all the rest is miscellaneous stuff.

**Comment by Accio1** [comment]
Is this better?
![Screenshot 2019-06-18 at 7 14 35 AM](https://user-images.githubusercontent.com/48496940/59677819-c6ed2e80-9198-11e9-8ac6-4f8cee18981d.png)

**Comment by NilsTheBest** [comment]
:+1:

**Comment by Accio1** [comment]
Great! We should probably get some more quotes before we start making the page on the actual site.

**Comment by NilsTheBest** [comment]
Woww, sorry I haven't checked this in so long. I guess after the exams I just switched to vacation mode. At this point, I think the quotes page is the last thing to do, right?

**Comment by Accio1** [comment]
Yes, for now. But we need more quotes before we can start it.

**Comment by Nambaseking01** [comment]
We need to start working on this, seems like this hasn't been discussed in a while,

**Comment by awesome5185** [comment]
oof only 4 (incl me) people commented on this page.

**Comment by Accio1** [comment]
Honestly, I'm not sure that this page is required. It would be a lot of work to maintain, and we need several more quotes to even start creating it.

**Comment by Accio1** [comment]
> Honestly, I'm not sure that this page is required. It would be a lot of work to maintain, and we need several more quotes to even start creating it.

@NilsTheBest what do you think?

**Comment by ghost** [comment]
> > Honestly, I'm not sure that this page is required. It would be a lot of work to maintain, and we need several more quotes to even start creating it.
> 
> @NilsTheBest what do you think?

It wouldn't be too much worse if more users contributed, and people were given roles. Also using Jekyll and encoding the data within a JSON document for the page to be auto-generated would help a lot.

**Comment by Accio1** [comment]
This conversation has gone stale. @NilsTheBest do you think that this page should still be implemented?

**Comment by Accio1** [comment]
Converting this to a discussion instead of an issue. If/when this idea gets more thoroughly planned, an issue will be created about the creation of the page.
